app/jobs/bulkrax/create_relationships_job.rb

```diff
@@ -107,9 +108,9 @@ def perform(parent_identifier:, importer_run_id: nil, run_user: nil) # rubocop:d
         # rubocop:enable Rails/SkipsModelValidations
 
         parent_entry&.set_status_info(errors.last, importer_run)
-
+        failure_count += 1
         # TODO: This can create an infinite job cycle, consider a time to live tracker.
-        reschedule(parent_identifier: parent_identifier, importer_run_id: importer_run_id)
+        reschedule(parent_identifier: parent_identifier, importer_run_id: importer_run_id, failure_count: failure_count) if failure_count < 5
```

Comment:
You are adding failure_count to the call to the reschedule method here, but didn't add it to the actual reschedule method.